supplementary/alica_ros1/alica_tests/libalica-tests/src/libalica-tests/plans/AdjacentSuccessMasterPlan.cpp

```diff
@@ -1,5 +1,5 @@
-#include <alica_tests/AdjacentSuccessMasterPlan.h>
 #include <alica_tests/TestWorldModel.h>
+#include <libalica-tests/plans/AdjacentSuccessMasterPlan.h>
```

Comment:
I think we can drop the lib prefix & use alica-tests instead

Reply:
we should also follow this folder structure for includes, where there is a sub-folder with the name of the library so that files are included as `#include <library_name/file_name.h>` in other places like turtlesim & alica standard library. That way there is less chance of name collisions

Reply:
can prevent duplicated folders i.e. move to alica_tests/include/alica_tests